dashboard/modules/job/job_head.py

```diff
@@ -85,6 +85,16 @@ async def get_job_info(self, job_id: str) -> JobDetails:
             else:
                 await self._raise_error(resp)
 
+    async def get_job_logs(self, job_id: str) -> str:
+        async with self._session.get(
+            self._address + f"/api/job_agent/jobs/{job_id}/logs"
+        ) as resp:
+            if resp.status == 200:
+                result_json = await resp.json()
+                return JobLogsResponse(**result_json).logs
```

Comment:
I'm not sure, but will the result be loaded into the memory all at once?

Reply:
My bad, I forgot to add API "GET /api/jobs/{job_or_submission_id}/logs/tail", this will return an iterator of job logs, it should be ok about memory.